src/components/four-column-block.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import Container from 'react-bootstrap/Container'
+import Card from 'react-bootstrap/Card'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+
+import arrow1 from '../images/arrow1.png'
+import arrow2 from '../images/arrow2.png'
+import arrow3 from '../images/arrow3.png'
+import arrow4 from '../images/arrow4.png'
+
+const FourColumnBlock = () => (
+  <Container>
+    <p className="lead text-center font-weight-bold">
+      As a result of this program, we anticipate an increase in:
+    </p>
+
+    <Row>
+      <Col sm={3}>
```

Comment:
Bootstrap generally expects columns to add up to 12. In this case, we have four elements, each takes up '3'. The 'sm' means on small devices these will break-down and stack on top of each other.